components/style/thread_state.rs

```diff
@@ -22,7 +22,7 @@ bitflags! {
     }
 }
 
-macro_rules! thread_types ( ( $( $fun:ident = $flag:ident ; )* ) => (
+macro_rules! thread_types (($( $fun:ident = $flag:ident ; )* ) => (
```

Comment:
I'm not sure whether the spaces should be fixed for this macro. It had a valid style, I believe?